shared/actions/platform-specific/index.desktop.js

```diff
@@ -142,6 +142,12 @@ const checkRPCOwnership = () =>
             })
           })
       )
+      yield Saga.put(
```

Comment:
Doesn't this need to be done conditionally? What if result.isOwner is false (the security hole the pipe check is meant to cover)?

Reply:
there is no catch handler on this promise, so when it throws the catch below catches and it goes to the error flow. you can see this if you just change the promise to
```
      yield Saga.call(
        () =>
          new Promise((resolve, reject) => {
            reject('MOCK')
          })
      )

```

you'll see the failedFatal handshake fires